bin/run-tests.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
 set -e
 
+# remove residue from previous runs
+find tests/*/ -name \*_analysis.json \( -not -name expected_analysis.json \) -delete
```

Comment:
```suggestion
find tests/ -name \*_analysis.json \( -not -name expected_analysis.json \) -delete
```

Would `find tests/` suffice?